CSharpMath.Forms.Example/CSharpMath.Forms.Example.UWP/CSharpMath.Forms.Example.UWP.csproj

```diff
@@ -171,9 +171,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms" Version="4.3.0.908675" />
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.2.8</Version>
-    </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.9" />
```

Comment:
why not `6.2.10` then?

Reply:
It updated during install of Xamarin.Essentials. Will modify to 6.2.10.